core/ArchiveProcessor/Loader.php

```diff
@@ -207,11 +217,22 @@ protected function loadArchiveData()
         // this hack was used to check the main function goes to return or continue
         // NOTE: $idArchives will contain the latest DONE_OK/DONE_INVALIDATED archive as well as any partial archives
         // with a ts_archived >= the DONE_OK/DONE_INVALIDATED date.
-        list($idArchives, $visits, $visitsConverted, $isAnyArchiveExists, $tsArchived, $value) = $this->loadExistingArchiveIdFromDb();
+        $archiveInfo = $this->loadExistingArchiveIdFromDb();
+        $idArchives = $archiveInfo['idArchives'];
```

Comment:
nice, this is definitely more readable 👍  Maybe for later could even great a data object for it. Not important for proof of concept though.